### PR TITLE
Add Example on Generating a Random Address and WIF for Alternative Network

### PR DESCRIPTION
**Pull request by jprichardson**
This PR adds an example on how to create a random keypair (address and WIF) for alternative networks such as Litecoin.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,9 +86,10 @@ This is because of the function-name-duck-typing used in [typeforce](https://git
 The below examples are implemented as integration tests, they should be very easy to understand.  Otherwise, pull requests are appreciated.
 
 - [Generate a random address](https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/test/integration/basic.js#L8)
-- [Generate a address from a SHA256 hash](https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/test/integration/basic.js#L20)
-- [Import an address via WIF](https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/test/integration/basic.js#L29)
-- [Create a Transaction](https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/test/integration/basic.js#L36)
+- [Generate a address from a SHA256 hash](https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/test/integration/basic.js#L19)
+- [Generate a address and wif for Litecoin](https://github.com/bitcoin/bitcoinjs-lib/blob/master/test/integration/basic.js#L29)
```
Comment (dcousens) [inline README.md]: > and **WIF**

Probably should capitalize the abbreviation

File: test/integration/basic.js [hunk 2]
```diff
@@ -26,6 +26,21 @@ describe('bitcoinjs-lib (basic)', function () {
     assert.equal(address, '1C7zdTfnkzmr13HfA2vNm5SJYRK6nEKyq8')
   })
 
+  it('can generate a random keypair for alternative networks', function () {
+    // for testing only
+    function rng () { return new Buffer('zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz') }
+
+    var litecoin = bitcoin.networks.litecoin
+
+    var keyPair = bitcoin.ECPair.makeRandom({ rng: rng })
```
Comment (dcousens) [inline test/integration/basic.js]: var keyPair = bitcoin.ECPair.makeRandom({ rng: rng, network: litecoin })

**Comment by jprichardson** [comment]
Squashed and fixed.

**Comment by dcousens** [comment]
Thanks @jprichardson. Only the integration tests were failing due to #399.
@coingeek this is of interest to you.

**Pull request merged**